### ISSUE TITLE
Raspbian packages

### ISSUE THREAD

**Issue by aidanharris** [presslabs/gitfs#275]
Would it be possible to get deb packages for raspbian?

I managed to install okay from source after building and installing [libgit2](https://github.com/libgit2/libgit2) from source using the [v0.24.6](https://github.com/libgit2/libgit2/tree/v0.24.6) tag.

It would be great if there was a binary package available though, especially since compiling on a Pi can be slow.

**Comment by narcisgarcia** [comment]
To compile/package gitFS is needed libgit2-dev (>= 0.24)
I could do this with Debian 9 as root:
```
apt install build-essential checkinstall automake python-virtualenv libffi-dev python3-dev libgit2-dev
wget https://github.com/PressLabs/gitfs/archive/master.zip
unzip master.zip
rm master.zip
mv gitfs-master gitfs
cd gitfs
# Fix subversion reference for virtualenv:
sed -i -e 's|python3.4|python3|g' Makefile
```
Write a "gitfs.spec" file with a line:
`Release: 1`
Write a "description-pak" file with a line:
`Version controlled file system`
Continue to compile and package:
```
make
checkinstall -D --fstrans --default --reset-uids "--maintainer=MyName" --pakdir=.. --pkgname=gitfs --requires=python3-minimal,python-fusepy,python-pygit2,python-setuptools --pkgrelease=1 --install=no make install
cd ..
rm -r gitfs
```
Anyway, package dependencies should be fixed for python-pygit2 (>= 0.24) because gitfs requires this version at least.

**Comment by vtemian** [comment]
We just published packages for Raspbian and updated the docs for that.
https://github.com/presslabs/gitfs/blob/master/docs/usage.md#installing
